src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenuItems-SignOut.cshtml

```diff
@@ -1,11 +1,22 @@
+@using OrchardCore.Admin
 @using OrchardCore.Users.Models
+@inject IAuthorizationService AuthorizationService
 
 @if (!Site.As<LoginSettings>().DisableLocalLogin)
 {
     <li>
-        <a class="dropdown-item" asp-area="@UserConstants.Features.Users" asp-controller="Account" asp-action="ChangePassword" asp-route-returnUrl="@FullRequestPath">
-            <i class="fa-solid fa-key fa-fw" aria-hidden="true"></i> @T["Change password"]
-        </a>
+        @if (await AuthorizationService.AuthorizeAsync(User, AdminPermissions.AccessAdminPanel))
```

Comment:
Why? this should be done from the driver. Look for existing `DisplayDriver<UserMenu>` implementation on how to do this.

Reply:
It could be, but it will requires two shapes IMHO

Reply:
I do think you need two shapes. If I recall correctly, I think we use different display types for the same shape. This allows you to create one template for admin and another for the front end.